### DISCUSSION TITLE
spaceduck theme for VisualStudio

### DISCUSSION THREAD

**Discussion by calvinhla** [pineapplegiant/spacedirt-theme#63] [Ideas]
Loving this theme. I found some [microsoft resources](https://github.com/microsoft/theme-converter-for-vs#getting-a-themes-json-file) to convert the VSCode theme you have to VisualStudio as well. Maybe you could include this in a readme or have a more built out respository for it if you had plans for this theme to be on visual studio as well. I'm also down to help set this up just not sure how you'd like it to be setup.

**Comment by pineapplegiant** [comment]
Hey @calvinhla, thanks for the love ❤️ 

I've admittedly been incredibly neglectful of Spaceduck ever since I got a job lol. But I'm totally down to help you port it to Visual Studio! 

If you would like to join the Spaceduck organization, we can do that, and you can create a repo under the "[spaceduck org](https://github.com/spaceduck-theme)". From there you can go ahead and create the repo and "maintain" the Spaceduck Visual Studio theme to make it "official". If that's too much, you're free to create a Spaceduck theme at will, and then I can link it to the readme no worries.

We do have a sort-of active discord community of like 5ish people that have done ports of spaceduck. I can also add you to it and we can chat there more candidly. We mostly talk about vim and color-schemes we like/dislike lol


I mostly just care about the (N)Vim theme and maintaining a good spec so others can inherit/create themes from that. I also really need to update the spec with a more expanded/updated palette, but that's another story.

**Reply by calvinhla** [reply]
Sure feel free to add me to the org + discord!

**Reply by pineapplegiant** [reply]
Send me your email/discord username?
